fabs/src/main/java/com/decathlon/vitamin/compose/fabs/VitaminFabs.kt

```diff
@@ -0,0 +1,128 @@
+import androidx.compose.material.icons.filled.Add
+import androidx.compose.material.ripple.LocalRippleTheme
+import androidx.compose.material.ripple.RippleTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.graphics.vector.rememberVectorPainter
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+object VitaminFabs {
+
+    /**
+     * The FAB must be used for primary, positive actions
+     * @param icon The icon to be displayed inside the FAB
+     * @param modifier The [Modifier] to be applied to the component
+     * @param enabled True if you can click on the FAB, otherwise false
+     * @param colors The colors of the background and the content in enabled and disabled
+     * @param sizes The sizes for the icon and the component itself
+     * @param ripple The ripple effect applied on the component
+     * @param onClick The callback to be called when the user click on the FAB
+     */
+    @Composable
+    fun Primary(
+        icon: Painter,
+        modifier: Modifier = Modifier,
+        enabled: Boolean = true,
+        colors: FabsColors = VitaminFabsColors.primary(),
+        sizes: FabSizes = VitaminFabSizes.default(),
+        ripple: RippleTheme = VitaminTheme.ripples.brand,
+        onClick: () -> Unit
+    ) {
+        CompositionLocalProvider(LocalRippleTheme provides ripple) {
+            FloatingActionButton(
+                interactionSource = if (enabled) {
+                    MutableInteractionSource()
+                } else {
+                    NoRippleInteractionSource()
+                },
+                elevation = if (enabled) {
+                    FloatingActionButtonDefaults.elevation()
+                } else {
+                    FloatingActionButtonDefaults.elevation(
+                        defaultElevation = 0.dp,
+                        pressedElevation = 0.dp,
+                        focusedElevation = 0.dp,
+                        hoveredElevation = 0.dp
+                    )
+                },
+                contentColor = if (enabled) {
+                    colors.contentColor
+                } else {
+                    colors.disabledContentColor
+                },
+                backgroundColor = if (enabled) {
+                    colors.backgroundColor
+                } else {
+                    colors.disabledBackgroundColor
+                },
+                modifier = modifier
+                    .size(sizes.size),
+                onClick = {
+                    if (enabled) {
+                        onClick()
+                    }
+                }
+            ) {
+                Icon(
+                    painter = icon,
+                    contentDescription = null,
```

Comment:
If this property is always `null`, your fab button is not accessible.
You should add a contentDescription as parameter or use api slotting scoped with a Vitamin object. Check other components like the progressbar.

Reply:
Probably the same comment for the extended version.